schedule/yam/test_cases/lvm+raid1_sle15.yaml

```diff
@@ -1,14 +1,13 @@
 ---
-name:           lvm+raid1@svirt-xen-pv
+name:           lvm+raid1
 description:    >
   Validation of partitioning for raid1 on lvm
   Installation of RAID1 using expert partitioner.
-vars:
-  YUI_REST_API: 1
 schedule:
   suggested_partitioning:
     - installation/partitioning/setup_raid1_lvm
+  security:
+  - installation/installation_settings/validate_ssh_service_enabled
+  - installation/installation_settings/open_ssh_port
```

Comment:
these two are not common to old, are default for some particular default schedule.

Reply:
As I see the `default_svirt-hyperv.yaml` contains both steps, but the related jobs don't mention any default schedule (they are inside `SLE_15_DEV`) If `SLE_15_DEV` is only for development and not for production we could safely remove both steps from the file. What do you think?

Reply:
yep, we gave up in some point with the hypervisor and reported to that squad, we used to have more jobs with that. I agree then, thx.

Reply:
Solved here 20562c20e4b65aedf39a85141813b548daaf03ab

Thanks